ee/apps/pdf-worker/src/templates/transcriptTemplate.tsx

```diff
@@ -0,0 +1,291 @@
+import React from 'react';
```

Comment:
Ignore most of this file, since it's a dummy template to check if processing works